commands/fun/trivia.js

```diff
@@ -0,0 +1,73 @@
+const Social = require(`${process.cwd()}/base/Social.js`);
+const Discord = require("discord.js");
+const html = require("html-entities").AllHtmlEntities;
+const snek = require("snekfetch");
+const h = new html();
+
+class Trivia extends Social {
+  constructor(client) {
+    super(client, {
+      name: "trivia",
+      description: "Delivers a quiz with the correct answer awarding 10 points.",
+      category: "Fun",
+      usage: "trivia",
+      extended: "Ready to wrack your brain? ",
```

Comment:
Is it intended to have a space inside the extended string?

Reply:
No, mistake in writing. Will be mended.